lib/zuora/objects/invoice.rb

```diff
@@ -21,14 +21,14 @@ class Invoice < Base
     validates_inclusion_of    :includes_recurring, :in => [true, false]
     validates_inclusion_of    :includes_usage, :in => [true, false]
     validates_length_of       :invoice_number, :maximum => 255  # String
-    validates_datetime_of     :last_email_sent_date, :allow_nil => true
+    # validates_datetime_of     :last_email_sent_date, :allow_nil => true
     validates_numericality_of :payment_amount, :allow_nil => true
-    validates_datetime_of     :posted_date, :allow_nil => true
+    # validates_datetime_of     :posted_date, :allow_nil => true
     validates_numericality_of :refund_amount, :allow_nil => true
     validates_inclusion_of    :status, :in => %w(Canceled Draft Error Posted), :allow_nil => true
-    validates_datetime_of     :target_date
+    # validates_datetime_of     :target_date
     validates_inclusion_of    :transferred_to_accounting, :in => %w(Processing Yes Error Ignore), :allow_nil => true
-    validates_datetime_of     :updated_date
+    # validates_datetime_of     :updated_date
```

Comment:
And do we know why these 4 validation rules are now commented out?

Reply:
Resolved here [04f281f](https://github.com/Sage/zuora/pull/18/commits/04f281f9a0a0b07961e1b1f4ebd5465beb8c27bc)